infrastructure/images/k6-action/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.21.3@sha256:a8560b36e8b8210634f77d9f7f9efd7ffa463e380b75e2e74aff4511df3ef88c
+FROM alpine:3.21.3@sha256:a8560b36e8b8210634f77d9f7f9efd7ffa463e380b75e2e74aff4511df3ef88c AS builder
 COPY --from=golang:1.24-alpine@sha256:7772cb5322baa875edd74705556d08f0eeca7b9c4b5367754ce3f2f00041ccee /usr/local/go/ /usr/local/go/
```

Comment:
Any reason for not just having `FROM golang:1.24-alpine` instead of copying from the golang base image?

Reply:
Nah, it's just a myopic view resulting from iterative nature of development.
Will update.

Reply:
Done!